External/Plugins/HaXeContext/Context.cs

```diff
@@ -577,6 +580,21 @@ public override void RemoveClassCompilerCache()
         }
         #endregion
 
+        #region SDK
+        private InstalledSDK GetCurrentSDK()
+        {
+            return hxsettings.InstalledSDKs.FirstOrDefault(sdk => sdk.Path == currentSDK);
```

Comment:
It can return null - for example when the list of installed SDKs is empty. The "Default" in the name refers to the fact that it returns `default(TSource)` (see [here](https://msdn.microsoft.com/en-us/library/vstudio/bb340482(v=vs.100).aspx)), which is `null` in this case.

Fun fact, this used to be a 5-line `foreach`-loop before Resharper suggested to turn it into a LINQ-expression. ;)